.cirrus.yml

```diff
@@ -791,6 +791,11 @@ podman_machine_windows_task:
           TEST_FLAVOR: "machine-hyperv"
     clone_script: *winclone
     main_script: ".\\repo\\contrib\\cirrus\\win-podman-machine-main.ps1"
+    always:
+        # Required for `contrib/cirrus/logformatter` to work properly
+        html_artifacts:
+            path: ./repo/*.html
```

Comment:
The logs look great! Thank you! But this spurious `/repo` breaks all existing conventions, and makes the logs impossible to find. Is there any way to remove the `/repo`?

Reply:
Sigh. Doesn't look like there's a way to do that [with Cirrus](https://cirrus-ci.org/guide/writing-tasks/#artifacts-instruction). Can the file be written (or moved) to `..`?

Reply:
Without a 1-click link the formatted logs are not really very useful. However, it's still incremental progress and maybe other Windows peeps will want it.